securedrop/tests/migrations/migration_15ac9509fc68.py

```diff
@@ -0,0 +1,211 @@
+# -*- coding: utf-8 -*-
+
+import random
+import string
+
+from datetime import datetime
+from sqlalchemy import text
+
+from db import db
+from journalist_app import create_app
+
+random.seed('ᕕ( ᐛ )ᕗ')
+
+
+def random_bool():
+    return bool(random.getrandbits(1))
+
+
+def bool_or_none():
+    return random.choice([None, True, False])
+
+
+def random_bytes(min, max, nullable):
+    if nullable and random_bool():
+        return None
+    else:
+        # python2 just wants strings, fix this in python3
+        return random_chars(random.randint(min, max))
+
+
+def random_username():
+    len = random.randint(3, 64)
+    return random_chars(len)
+
+
+def random_chars(len, chars=string.printable):
+    return ''.join([random.choice(chars) for _ in range(len)])
+
+
+def random_datetime(nullable):
```

Comment:
These functions are going to be used a _lot_ in migration testing - at some point we'll want to move these functions to a utility file